server/src/internalClusterTest/java/org/opensearch/discovery/DiscoveryDisruptionIT.java

```diff
@@ -166,7 +166,7 @@ public void testElectClusterManagerWithLatestVersion() throws Exception {
         logger.info("--> forcing a complete election to make sure \"preferred\" cluster-manager is elected");
         isolateAllNodes.startDisrupting();
         for (String node : nodes) {
-            assertNoMaster(node);
+            assertNoClusterManager(node);
```

Comment:
`getMasterService`

Reply:
https://github.com/opensearch-project/OpenSearch/blob/2.0.0-rc1/server/src/main/java/org/opensearch/cluster/service/ClusterService.java#L216 is a public method in `server/src/main` directory. I will address them in a following PR and target for version 3.0